sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/python/GpuArrowPythonRunner.scala

```diff
@@ -188,6 +189,14 @@ trait GpuPythonArrowOutput extends Arm { _: GpuPythonRunnerBase[_] =>
   }
 }
 
+trait DefaultTableToBatchConverter {
+  def pythonOutSchema: StructType
+
+  def toBatch(table: Table): ColumnarBatch = {
```

Comment:
No, extending GpuPythonArrowOutput requires it being a GpuPythonRunnerBase.
And DefaultTableToBatchConverter is designed to avoid the boilerplate code of `toBatch` in the children.

Since this is not a good design, just move the default `toBatch` method to children instead.